Cake.Issues.Recipe/Content/build.cake

```diff
@@ -113,16 +115,31 @@ IssuesBuildTasks.CreateFullIssuesReportTask = Task("Create-FullIssuesReport")
         IssuesParameters.OutputDirectory.CombineWithFilePath(reportFileName);
     EnsureDirectoryExists(IssuesParameters.OutputDirectory);
 
+    IIssueReportFormat issueFormat = null;
+
+    if (!Context.Environment.Runtime.IsCoreClr)
+    {
+        Information("Loading Generic Reporter");
+        var genericReporter = new GenericReporterData(Context);
+        var settings = genericReporter.FromEmbeddedTemplate(genericReporter.GenericIssueReportTemplateEnum("HtmlDxDataGrid"));
+        genericReporter.WithOption(settings, genericReporter.HtmlDxDataGridOptionEnum("Theme"), genericReporter.DevXtremeEnum("MaterialBlueLight"));
+
+        issueFormat = genericReporter.GenericIssueReportFormat(settings);
+    }
+
     // Create HTML report using DevExpress template.
-    var settings =
-        GenericIssueReportFormatSettings
-            .FromEmbeddedTemplate(GenericIssueReportTemplate.HtmlDxDataGrid)
-            .WithOption(HtmlDxDataGridOption.Theme, DevExtremeTheme.MaterialBlueLight);
-    CreateIssueReport(
-        data.Issues,
-        GenericIssueReportFormat(settings),
-        data.RepositoryRootDirectory,
-        data.FullIssuesReport);
+    if (issueFormat != null)
```

Comment:
Going forward, there won't be a single report format but multiple. It might make perfectly sense for example to create an HTML and SARIF report.

Reply:
True, but currently only HTML report is supported, but I can change everything to be a List of IIssueReportFormat instead.